shell/platform/android/hardware_buffer_external_texture_gl.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_HARDWARE_BUFFER_EXTERNAL_TEXTURE_GL_H_
+#define FLUTTER_SHELL_PLATFORM_HARDWARE_BUFFER_EXTERNAL_TEXTURE_GL_H_
+
+#include <mutex>
+
+#include "flutter/impeller/renderer/backend/gles/gles.h"
+
+#include <android/hardware_buffer.h>
+#include <android/hardware_buffer_jni.h>
+
+#include "flutter/common/graphics/texture.h"
+#include "flutter/fml/logging.h"
+#include "flutter/shell/platform/android/android_context_gl_skia.h"
+#include "flutter/shell/platform/android/platform_view_android_jni_impl.h"
+
+namespace flutter {
+
+class HardwareBufferExternalTextureGL : public flutter::Texture {
+ public:
+  explicit HardwareBufferExternalTextureGL(
+      const std::shared_ptr<AndroidContextGLSkia>& context,
+      int64_t id,
+      const fml::jni::ScopedJavaGlobalRef<jobject>&
+          hardware_buffer_texture_entry,
+      const std::shared_ptr<PlatformViewAndroidJNI>& jni_facade);
+  ~HardwareBufferExternalTextureGL() override;
+
+  // Implementing Texture.
```

Comment:
The convention in the engine is to [add a comment like so for overrides](https://github.com/flutter/engine/blob/c00bcae9f1da9623444b5299095adf803da37f3d/shell/common/shell.h#L550):

```
// |impeller::Texture|
```

Reply:
Done.

Good to know.